test-utils/src/indexer/test_indexer.rs

```diff
@@ -139,6 +142,30 @@ impl<R: RpcConnection + Send + Sync + 'static> Indexer<R> for TestIndexer<R> {
         Err(IndexerError::Custom("Merkle tree not found".to_string()))
     }
 
+    async fn get_subtrees(
```

Comment:
```suggestion
    async fn get_sub_trees(
```
feels more intuitive tbh

Reply:
hm why? "subtree" is a standard term in graph theory